docs/en_US/Compressor/Framework.md

```diff
@@ -52,7 +58,7 @@ A `pruner` is responsible for:
 3. Use `weight masker` to calculate masks of layers while pruning.
 4. Export pruned model weights and masks.
 
-## Implement a new pruning algorithm
+### Implement a new pruning algorithm
 
 Implementing a new pruning algorithm requires implementing a `weight masker` class which shoud be a subclass of `WeightMasker`, and a `pruner` class, which should a subclass `Pruner`.
```

Comment:
should a subclass -> should be a subclass of

Reply:
good catch, thanks!